provisioner/k8s/manifests/01_cluster_role.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Is all the rbac generated with `controller-gen` from[ `+kubebuilder` markers in the controllers](https://github.com/operator-framework/rukpak/blob/79f1e29557df669bb12dc5bef9d222f3c14c7c55/provisioner/k8s/controllers/bundle_controller.go#L66-L70)? If not, did we make a conscious decision to not use that?

Reply:
Also, doesn't the k8s provisioner need `*`/`*`/`*` permissions in order to be able to deploy arbitrary manifests?

Reply:
This is what I was thinking about when writing the RBAC. I wanted to avoid `"*"` on everything at the very beginning, since the actual set of permissions required to deploy an operator via a plain k8s bundle is limited to the realistic scope of the manifests in the bundle: things in the core API group, RBAC, CRDs, and maybe some additional basic types. 

If it's possible, we should give it the smallest set of permissions at the beginning, and then expand. If it's possible to avoid https://github.com/operator-framework/operator-lifecycle-manager/issues/1685 up front we should.

Reply:
> Is all the rbac generated with `controller-gen` from[ `+kubebuilder` markers in the controllers](https://github.com/operator-framework/rukpak/blob/79f1e29557df669bb12dc5bef9d222f3c14c7c55/provisioner/k8s/controllers/bundle_controller.go#L66-L70)? If not, did we make a conscious decision to not use that?

We should use those if possible, right now the RBAC is done manually -- it looks like the markers weren't updated with the new `core.rukpak.io` label though, which is interesting